Source/WebCore/css/CSSProperties.json

```diff
@@ -11286,6 +11286,14 @@
                 "category": "scroll-animations",
                 "url": "https://drafts.csswg.org/scroll-animations-1/#propdef-view-timeline-inset"
             }
+        },
+        "<anchor-size>": {
```

Comment:
Not sure if I should define `<anchor-size>` like this, then use `CSSPropertyParsing::consumeAnchorSize`, or should I just use `CSSPropertyParserHelpers::consumeIdentRaw`

Reply:
Consensus is to use `CSSPropertyParserHelpers::consumeIdentRaw`